### DISCUSSION TITLE
why does it require network permissions?

### DISCUSSION THREAD

**Discussion by unusualevent** [d4rken-org/sdmaid-se#1253] [Q&A]
**Describe the bug**

**To Reproduce**
1. go to https://f-droid.org/en/packages/eu.darken.sdmse/
2. look at the permissions.

**Expected behavior**
There should be only the permissions necessary to complete the process

**Smartphone (please complete the following information):**
 - Device: Pixel 6
 - OS: Android
 - Root status - not rooted
 - SD Maid 2/SE version 1.1.2-rc

**Comment by d4rken** [comment]
Internet permission is used for update checks and the MOTD system (message system if something happens and I want to send everyone an in-app info).

Both is optional and the user can opt-in/out during on-boarding.

**Reply by unusualevent** [reply]
I did not see a place to opt out of it during onboarding.

**Reply by d4rken** [reply]
![Screenshot_20240701_115303](https://github.com/d4rken-org/sdmaid-se/assets/1439229/4cca97fe-aafe-40fa-9b51-fefd371b8e48)

**Comment by MarkyMarkDE** [comment]
Right, it needs this permission for update check and sometimes important messages from the Dev, this can't work without this permission but you can disable both in the app.
